.github/workflows/build.yml

```diff
@@ -67,10 +67,7 @@ jobs:
 
       - uses: actions/setup-python@v5
         with:
-          python-version: |
-            ${{ matrix.group == 'storage' && '3.10' ||
```

Comment:
I think we should still run something on 3.10.

Reply:
What about adding a 3.10 job for the release branch?

Reply:
Whole CI is a lot. I think just running storage on 3.10 is enough.

Reply:
I mean we only run it on the release branch again and it's not that expensive. I added the whole CI on the release branch as a lower bound sanity check. I think it makes sense.